scalameter-core/src/main/scala/org/scalameter/picklers/containers.scala

```diff
@@ -6,7 +6,7 @@ import java.util.Date
 import Implicits._
 
 
-abstract class TraversablePickler[C[_] <: Traversable[_], T: Pickler](implicit cbf: CanBuildFrom[Nothing, T, C[T]]) extends Pickler[C[T]] {
+abstract class TraversablePickler[C[_] <: Traversable[_], T: Pickler](implicit @transient cbf: CanBuildFrom[Nothing, T, C[T]]) extends Pickler[C[T]] {
```

Comment:
Having `@transient` here could cause troubles later.

The key issue is, it seems, that pickler to easily becomes a part of the object state.
This breaks `SerializationPersistor`. I would suggest that, instead of making `Pickler`s a part of the `Key` state, to just embed their classnames into the `Key`. The `Pickler` can then be instantiated via reflection, or looked up from some lookup table.